### ISSUE TITLE
Multi-lingual Detection Model Training dataset

### ISSUE THREAD

**Issue by JoeyTribbiani77** [PaddlePaddle/PaddleOCR#8466]
Hi Everyone,

I wanted to fine-tune the Multi-lingual Detection Model on my data but I don’t want to lose its current performance

So I wanted to have access to the original training data, would it be hard to provide a link to the used data in English and Multi-lingual training?

**Comment by tink2123** [comment]
Sorry, we are unable to provide training data for the time being, you can use open source data + custom data, and use small learning rate to finetune.
